cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -53,7 +57,8 @@ impl PluginSettings for EdgeAddRemovePlugin {
 ///     2. next
 /// 2. *.remove
 ///     1. previous
```

Comment:
Seems both `previous` and `previous_regex` data would be applied. Shouldn't we prefer `previous_regex` instead?

Reply:
I think the existing syntax can coexist with the additional one, so I don't see why we should remove the existing one. What makes you think we should?

Reply:
I'd expect we use either `previous` or `previous_regex` in `blocked-edges`, but current implementation allows both. This may lead to a few odd situations - lets throw error when both are specified?

Reply:
I don't see why it would be an error. They could be used in conjunction as well. Why do you want to restrict it?

Reply:
Its not an error really, just feels that it might have undesirable consequences. Maybe I'm overreacting really as I can't come up with a potential issue on this one

Reply:
If it helps, I don't see a potential issue in the code either, and I've thought about it quite a bit :slightly_smiling_face: